manager/usermanager/createUser.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 // CreateUser get the user information
```

Comment:
Please update the godoc to give away helpful hints about the function. Do mention what is expected from it if `isSignup` is true/false.